Dockerfile

```diff
@@ -71,4 +71,4 @@ RUN mkdir /bzt-configs \
   && rm -rf /tmp/*
 
 WORKDIR /bzt-configs
-CMD bzt -l /tmp/artifacts/bzt.log /bzt-configs/*.yml
+ENTRYPOINT ["sh", "-c", "bzt /bzt-configs/*.yml -l /tmp/artifacts/bzt.log \"$@\"", "ignored"]
```

Comment:
I did some more thinking after my last comment, specifically about this statement:

> Getting rid of *.yml command also simplifies mounting part of Docker invocation and makes usage pattern more natural.

Maybe I misunderstood, but I thought you were saying we could get rid of the need to pass something like `-v /tmp/my-test:/bzt-configs` to the `docker run` command. The trouble with that is the ENTRYPOINT is just a json array of strings, and I believe any additional command line arguments are just appended as strings to that array. So we still need to mount a volume to get the actual configuration files into Docker.

As I see it, there are four options:

1. Update ENTRYPOINT as I have it now to also accept .json configuration files. So the ENTRYPOINT would look something like this:

        ENTRYPOINT ["sh", "-c", "bzt /bzt-configs/*.yml /bzt-configs/*.json -l /tmp/artifacts/bzt.log \"$@\"", "ignored"]

    Doing so would mean all tests would always run. To my understanding, any command line overrides would be applied to all of them, so naming collisions could lead to unexpected results.
2. Move the *.yml and any other defaults to a CMD to [act as default additional arguments to the ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#/exec-form-entrypoint-example). That would look something like this:

        ENTRYPOINT ["sh", "-c", "bzt -l /tmp/artifacts/bzt.log \"$@\"", "ignored"]
        CMD ["/bzt-configs/*.yml", "/bzt-configs/*.json"]

    So by default everything should work the same, but the user can still pass in a subset of config file names to be run instead. My understanding is that any additional command line arguments the user adds will override what's in CMD, so if they want to use a Taurus command line override they will *also* have to specify which test to run as the default will not be used.
3. Remove the *.yml completely, but leave the `"sh"` command in the ENTRYPOINT. That would look something like this:

        ENTRYPOINT ["sh", "-c", "bzt -l /tmp/artifacts/bzt.log \"$@\"", "ignored"]

    That would mean the user **has** to provide the list of config file names to run, but still could use a wildcard to run all configs in the mounted volume.
4. Simplify the ENTRYPOINT as much as possible by removing the *.yml *and* the `"sh"` and related strings. That would look something like this:

        ENTRYPOINT ["bzt -l /tmp/artifacts/bzt.log"]

    Not only would the user **have** to provide the list of config file names to run, they would **not** be able to use wildcards or any other shell-interpreted variables.

That's my assessment. Please let me know which option seems best to you, or if I'm misunderstanding something entirely!